plato/draw/fresnel/Box.py

```diff
@@ -0,0 +1,7 @@
+from ... import draw
+from .Lines import Lines
+from ..internal import ShapeDecorator
+
+@ShapeDecorator
```

Comment:
@klarh Do we need to have this decorator on all shape classes for all backends? It's missing in a lot of places. I can make a separate PR if that is needed.

Reply:
Off the top of my head, it is only needed if the docstring needs to be updated (i.e. if we are changing _ATTRIBUTES).